lib/galaxy/datatypes/media.py

```diff
@@ -32,6 +33,94 @@ def ffprobe(path):
     return data["format"], data["streams"]
 
 
+magic_number = {
+    "mp4": {
+        "offset": 4,
+        "hex": [
+            "66 74 79 70 69",
+            "66 74 79 70 6D",
+            "66 74 79 70 4D",
+        ],
+    },
+    "flv": {"offset": 0, "hex": ["46 4C 56 01"]},
+    "mkv": {"offset": 0, "hex": ["1A 45 DF A3"]},
+    "webm": {"offset": 0, "hex": ["1A 45 DF A3"]},
+    "mov": {"offset": 4, "hex": ["66 74 79 70 71", "6D 6F 6F 76"]},
+    "wav": {"offset": 8, "hex": ["57 41 56 45"]},
+    "mp3": {
+        "offset": 0,
+        "hex": [
+            "49 44 33",
+            "FF E0",
+            "FF E1",
+            "FF E2",
+            "FF E3",
+            "FF E4",
+            "FF E5",
+            "FF E6",
+            "FF E7",
+            "FF E8",
+            "FF E9",
+            "FF EA",
+            "FF EB",
+            "FF EC",
+            "FF ED",
+            "FF EE",
+            "FF EF",
+            "FF F0",
+            "FF F1",
+            "FF F2",
+            "FF F3",
+            "FF F4",
+            "FF F5",
+            "FF F6",
+            "FF F7",
+            "FF F8",
+            "FF F9",
+            "FF FA",
+            "FF FB",
+            "FF FC",
+            "FF FD",
+            "FF FE",
+            "FF FF",
+        ],
+    },
+    "ogg": {"offset": 0, "hex": ["4F 67 67"]},
+    "wma": {"offset": 0, "hex": ["30 26 B2 75"]},
+    "wmv": {"offset": 0, "hex": ["30 26 B2 75"]},
+    "avi": {"offset": 8, "hex": ["41 56 49"]},
+    "mpg": {
+        "offset": 0,
+        "hex": [
+            "00 00 01 B0",
+            "00 00 01 B1",
+            "00 00 01 B3",
+            "00 00 01 B4",
+            "00 00 01 B5",
+            "00 00 01 B6",
+            "00 00 01 B7",
+            "00 00 01 B8",
+            "00 00 01 B9",
+            "00 00 01 BA",
```

Comment:
are these all seen in practice? Wikipedia's list suggests it should be a much reduced set

```
00 00 01 BA
00 00 01 B3 
```

Reply:
I was using this source:
https://www.garykessler.net/library/file_sigs.html

> 00 00 01 Bx 	  	....
> MPEG, MPG 	  	MPEG video file
> Trailer:
> 00 00 01 B7 (...·)